src/app/pages/resources/dashboard/dashboard.jsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Card,
+  CardContent,
+  Typography,
+  Container,
+  Button,
+  makeStyles,
+} from '@material-ui/core';
+import { useFetchReportUrl } from 'api/reports';
+import { palette } from 'themes';
+
+import AssessmentOutlinedIcon from '@material-ui/icons/AssessmentOutlined';
+import DashboardOutlinedIcon from '@material-ui/icons/DashboardOutlined';
+import GroupOutlinedIcon from '@material-ui/icons/GroupOutlined';
+import HighlightOutlinedIcon from '@material-ui/icons/HighlightOutlined';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
```suggestion
const useStyles = makeStyles({
```

Passing a function into the makeStyles was what we did early on, but is not the standard going forward